packages/aws-cdk-lib/aws-dynamodb/lib/table-v2.ts

```diff
@@ -992,4 +1009,20 @@ export class TableV2 extends TableBaseV2 {
       throw new Error(`You may not provide more than ${MAX_LSI_COUNT} local secondary indexes`);
     }
   }
+
+  private validatePitr(props: TablePropsV2) {
+    const recoveryPeriodInDays = props.pointInTimeRecoverySpecification?.recoveryPeriodInDays;
+
+    if (props.pointInTimeRecovery !== undefined && props.pointInTimeRecoverySpecification !== undefined) {
+      throw new Error('`pointInTimeRecoverySpecification` and `pointInTimeRecovery` are set. Use `pointInTimeRecoverySpecification` only.');
```

Comment:
We're replacing any validation error from `new Error` to `new ValidationError`. Can you stick to the new validation error please?